process/process.go

```diff
@@ -209,6 +218,33 @@ func loadProcess(ctx context.Context, pid int) (*Process, error) {
 	}
 	process.ParentPid = int(ppid)
 
+	// Get all environment variables
+	envVariables, err := pInfo.EnvironWithContext(ctx)
+
+	// Not important enough to stop execution of the function
+	if err == nil {
+		// Try to find PORTMASTER_PROFILE variable
+		for _, envVar := range envVariables {
+			if strings.HasPrefix(envVar, profileEnvironmentVariable) {
```

Comment:
Should we be matching for `PORTMASTER_PROFILE=`?
Right now, this would match anything starting with `PORTMASTER_PROFILE`.